mass_editing/__manifest__.py

```diff
@@ -5,6 +5,7 @@
     'version': '11.0.1.0.0',
     'author': 'Serpent Consulting Services Pvt. Ltd., '
               'Tecnativa, '
+              'brain-tec AG, '
```

Comment:
For this newest version, I don't consider your contribution to be upgraded to co-authorship. Please keep only the contributor in the README part.

Reply:
Ok, I understand this. Thanks for the suggestion